test/help-create-config/expected-stdout

```diff
@@ -0,0 +1,23 @@
+Command "create config"
+
+Creates a new config in the local directory structure.
+A new unique ID is assigned to the new object (there is no need to call "persist").
+To save the new object to the project, call "push" after "create".
+
+You will be prompted for the object specification,
```

Comment:
trocha mi tu pride nejasna vazba, ze neni specifikovane co znamena objekt, ajked asi z kontextu jasne vyplyva ze to je config alebo configrow. Nechal by som to, ten wording sa ustali po beta testingu.

Reply:
ok, toto vymenim ... wording sa bude este riesit aj s Hankou